app/pages/organization/stats/components/ProjectStats/components/Workflows/components/ActionText.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint func-names: off, prefer-arrow-callback: off */
+import React from 'react';
+import Translate from 'react-translate-component';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import ActionText from './ActionText';
+
+describe('ActionText', function () {
+  let wrapper;
+  const handleClickSpy = sinon.spy();
+
+  before(function () {
+    wrapper = shallow(
+      <ActionText
+        content='organization.stats.expandWorkflowStats'
+        handleClick={handleClickSpy}
+      />
+    );
+  });
+
+  it('should render without crashing', function () {});
```

Comment:
Same here as the comment before. It seems a bit awkward to me to have an empty test block. There are a few more examples of this in these tests.